src/form/Input/InputStyle.ts

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import { themeColor, themeSpacing } from '@amsterdam/asc-ui';
+import { INPUT_SIZE } from '../../shared/constants';
```

Comment:
Moet bepaald worden door het design systeem denk ik.

Reply:
Taken from the asc code. My intention was to copy as much as the asc approach to make it easier moving a component to asc beta. 

I would prefer CSS variables inside a theme.css